site/content/contribute/mobile/push-notifications/corporate-proxy.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Corporate Proxy Server"
+date: 2020-03-09T11:35:32
+weight: 3
+subsection: "Setup Push Notifications"
+---
+
+### How do I receive mobile push notification if my IT policy requires the use of a corporate proxy server?
+
+When your IT policy requires a corporate proxy to scan and audit all outbound traffic the following options are available:
+
+###### 1 - Deploy Mattermost with connection restricted post-proxy relay in DMZ or a trusted cloud environment
+
+Some legacy corporate proxy configurations may be incompatible with the requirements of modern mobile architectures, such as the requirement of HTTP/2 requests from Apple to send push notifications to iOS devices.
+
+In this case, a [post-proxy relay](https://docs.mattermost.com/overview/faq.html#what-are-pre-proxy-and-post-proxy-relays) can be deployed to take messages from the Mattermost server passing through your corporate IT proxy in the incompatible format, e.g. HTTP/1.1, transform it to HTTP/2 and relay it to its final destination, either to the [Apple Push Notification Service (APNS)](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1) and [Google Fire Cloud Messaging (FCM)](https://firebase.google.com/docs/cloud-messaging) services. 
```

Comment:
Instead of taking the admin to docs for the definition of a post-proxy relay, let define it in this section

Reply:
afaik the sentence describes the functionality of a post-proxy relay, the link only exists to reference it to where it was first introduced, sort of like a wiki article linking a buzzword.

Reply:
True. I wonder if we can simplify the sentence to read `In this case, a proxy can be deployed to ...`

Reply:
^let's do the simplification as long as it keeps the text accurate

Reply:
I'm at a loss of knowledge here, whom would I ask if this will keep the text accurate?

Reply:
@faase For this iteration, we can keep this as. It includes the definition in brackets, so it's technically accurate. We can simplify later if needed

```
In this case, a **post-proxy relay** (which accepts network traffic from a corporate proxy such as NGINX, and transmits it to the final destination) can be deployed to take messages from the Mattermost server passing through your corporate IT proxy in the incompatible format, e.g. HTTP/1.1, transform it to HTTP/2 and relay it to its final destination, either to the [Apple Push Notification Service (APNS)](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/APNSOverview.html#//apple_ref/doc/uid/TP40008194-CH8-SW1) and [Google Fire Cloud Messaging (FCM)](https://firebase.google.com/docs/cloud-messaging) services. 
```